core/vm/interface.go

```diff
@@ -47,6 +47,9 @@ type StateDB interface {
 	GetState(common.Address, common.Hash) common.Hash
 	SetState(common.Address, common.Hash, common.Hash)
 
+	GetBalanceLittleEndian(address common.Address) []byte
```

Comment:
yeah it would be nice not to do that, because the data should be stored in the DB in the exact same format that it needs to be read.

Reply:
It might be stored in little-endian on disk.  But `StateDB.GetBalance` returns a `big.Int` whose `Bytes()` method returns the bytes in big-endian format.  So a conversion method is needed for this one afaict.

Reply:
:facepalm: actually it might be even worse: if it's BE in the db, then the `big.Int` value is probably a LE number interpreted as a BE number... In that case, you still don't need these functions but it's a dangerous game to play because all calculations will be wrong.

Come to think of it, _is it_ using LE in the DB? :thinking: Gotta check that.